css/_custom.css

```diff
@@ -8,11 +8,16 @@
 }
 
 @media (--breakpoint-not-small) {
-  .h-6-ns { height: 24rem; }
+  .h6-ns { height: 24rem; }
 }
 
 
 
+/* Display */
+@media (--breakpoint-not-small) {
+    .dn-ns { display: none !important; }
```

Comment:
In tachyons the `dn` class does not have an `!important`. Why do you need it for the media query?